web3/utils/contracts.py

```diff
@@ -41,6 +46,13 @@
 )
 
 
+class FunctionNotSpecified(Exception):
```

Comment:
Can this be dropped entirely? I don't see the usage.

Reply:
Good find!